### PR TITLE
Better skeleton boilerplate.

### PR DESCRIPTION
**Pull request by lucalanca**
# Styles

— `styles` folder no longer exist.
— style files should now be under: `styles` (color, typography, sizes, etc), `elements` (utilities, similar to 4.layout or 3.base or the objects layer on ITCSS) or `modules`
— the "root" css file is placed on templates. it is also possible to add page-dependent styles by placing more root files under /pages
# Scripts

— `scripts` folder no longer exist.
— the "root" js file and the "vendor" js files are now under "/templates/default" but one could easily create page specific files by creating a file under /pages
# Styles

— added typography
— added colors
— added spacings
# Elements

— added buttons
— added containers
— added grid
# Modules

— moved header to modules.

### CODE REVIEW

**Comment by meodai** [comment]
I like it! But default button styles? Why does not feel like this should be part of skeleton. But rather be a separate thing.

**Comment by ingvijonasson** [comment]
Is the idea behind Skeleton to be as bare as possible? @meodai is that maybe the reason why you ask about button styles?

**Comment by meodai** [comment]
@ingvijonasson yes. To me a boilerplate should not go further then resetting or normalizing. But i like the idea of very reusable components maybe something like skeletton-bootstrap? Or meat :D

File: src/styles/colors/style.scss [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    violet         : #D0A9F6,
+  )
+);
+
+// @import "bonescss/mixins";
+// @TODO remove once bonescss supports nested maps and rename c(...) to color(...)
+/**
+ * Color functions
+ */
+@function c($palette, $name) {
```
Comment (meodai) [inline src/styles/colors/style.scss]: this is part of sensible already https://github.com/meodai/bonescss/blob/master/mixins/_tools-color.scss

**Comment by peterbraden** [comment]
Are js files glob included? How is ordering done?

**Comment by lucalanca** [comment]
@peterbraden: no, that's still as it was before. Only two root files: vendor and main. If something needs to be included, it needs to be explicitly imported there. (vendor is uses CommonChunks plugin to create a different bundle for vendor scripts, that can be cached as they don't often change).

File: src/styles/colors/style.scss (hunk 1 above)
Comment (lucalanca) [inline src/styles/colors/style.scss]: @meodai this function on bonescss doesn't support 2 level deep maps.

Comment (meodai) [inline src/styles/colors/style.scss]: @lucalanca feel free to do a pull request there ;)

File: src/modules/header/layout.jade [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      all: 'one-whole',
+      mobile: 'one-whole'},
+    ratio: {
+      all: 'landscape'},
+  })
+    h1.t-h1= data.title
+    h2= data.subtitle
+    .l-grid
+      .l-grid-cell
+        a.btn(href="https://github.com/ginetta/skeleton") Github
```
Comment (lejoe) [inline src/modules/header/layout.jade]: Can we move the content to the data?

**Comment by lucalanca** [comment]
replaced by https://github.com/ginetta/skeleton/pull/148

**Pull request closed without merging**